docs/api/web-contents.md

```diff
@@ -1264,6 +1264,9 @@ Returns [`PrinterInfo[]`](structures/printer-info.md).
   * `dpi` Object (optional)
     * `horizontal` Number (optional) - The horizontal dpi.
     * `vertical` Number (optional) - The vertical dpi.
+  * `headerFooterInfo` Object (optional)
```

Comment:
Why is an object needed here?  I think it would be more straightforward as:
```suggestion
  * `header` String - String to be printed as page header.
  * `footer` String - String to be printed as page footer.
```

Reply:
This was done to simplify logic: header and footer can only be set if `printing::kSettingHeaderFooterEnabled` is set to `true`; and the methodology for this involving the least conditional checking was implicitly checking for the object itself. If an object was passed then both header _and_ footer must be set, and so we can proceed slightly more cleanly :)

Reply:
@codebytere maybe I missed this, but what if you only want to set header and not footer or vice versa?

Reply:
hmm, i think i might be able to rework the logic a bit to pass empty strings, although i think Chromium intends for both to be set if any are set 🤔

Reply:
@jkleinsc all set ⭐️